roles/elasticsearch/tests/inventory

```diff
@@ -0,0 +1,2 @@
+localhost
```

Comment:
What's this `tests` folder for?

Reply:
good catch, I didn't notice before, they have been created automatically by running `molecule init role`, will try to remove them